.travis.yml

```diff
@@ -4,8 +4,19 @@ scala:
   - 2.11.8
   - 2.12.1
 
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot
```

Comment:
@jvican It's better (and I believe safe) to cache the whole ~/.sbt directory. That way you cache the launcher in ~/.sbt/launchers